### PR TITLE
add chinese readme.md

### PR DESCRIPTION
**Pull request by quincylin1**
added chinese readme.md

### CODE REVIEW

**Comment by innerlee** [comment]
Linting failed.

File: README_zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+请参考[快速入门](docs/getting_started.md)文档学习 MMOCR 的基本使用。 
+
+## 贡献指南
+
+我们感谢所有的贡献者为改进和提升 MMOCR 所作出的努力。请参考[贡献指南](.github/CONTRIBUTING.md)来了解参与项目贡献的相关指引。
+
+## 致谢
+
+MMOCR 是一款由来自不同高校和企业的研发人员共同参与贡献的开源项目。我们感谢所有为项目提供算法复现和新功能支持的贡献者，以及提供宝贵反馈的用户。 我们希望此工具箱和基准测试可以通过提供灵活的工具箱来重新实现现有方法并开发用户自建的新OCR方法，从而为不断发展的研究社区服务。
```
Comment (jeffreykuang) [inline README_zh-CN.md:104]:
```suggestion
MMOCR 是一款由来自不同高校和企业的研发人员共同参与贡献的开源项目。我们感谢所有为项目提供算法复现和新功能支持的贡献者，以及提供宝贵反馈的用户。 我们希望此工具箱可以帮助大家来复现已有的方法和开发新的方法，从而为研究社区贡献力量。
```

File: README_zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+[![build](https://github.com/open-mmlab/mmocr/workflows/build/badge.svg)](https://github.com/open-mmlab/mmocr/actions)
+[![docs](https://readthedocs.org/projects/mmocr/badge/?version=latest)](https://mmocr.readthedocs.io/en/latest/?badge=latest)
+[![codecov](https://codecov.io/gh/open-mmlab/mmocr/branch/main/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmocr)
+[![license](https://img.shields.io/github/license/open-mmlab/mmocr.svg)](https://github.com/open-mmlab/mmocr/blob/main/LICENSE)
+[![PyPI](https://badge.fury.io/py/mmocr.svg)](https://pypi.org/project/mmocr/)
+[![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/open-mmlab/mmocr.svg)](https://github.com/open-mmlab/mmocr/issues)
+[![Percentage of issues still open](https://isitmaintained.com/badge/open/open-mmlab/mmocr.svg)](https://github.com/open-mmlab/mmocr/issues)
+
+MMOCR是基于PyTorch和mmdetection的开源工具箱，专注于文本检测，文本识别以及相应的下游任务，如关键信息提取。 它是OpenMMLab项目的一部分。
```
Comment (innerlee) [inline README_zh-CN.md:17]: space between english and chinese

File: README_zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+<div align="left">
+  <img src="resources/illustration.jpg"/>
+</div>
+
+### 主要特性
+
+-**全流程**
+
+   该工具箱不仅支持文本检测和文本识别，还支持其下游任务，例如关键信息提取。
```
Comment (innerlee) [inline README_zh-CN.md:31]: ~~该工具箱~~

**Pull request closed without merging**